docs/Guides/Scim/Guide-to-sync-users-in-vendasta.md

```diff
@@ -0,0 +1,42 @@
+# Guide to use Vendasta SCIM APIs to sync Users
+
+
+## Overview
+System for Cross-domain Identity Management ([SCIM](https://en.wikipedia.org/wiki/System_for_Cross-domain_Identity_Management)) focuses on syncing user accounts and permissions between systems but has an extension system that allows syncing any type of record. There are several well known endpoint paths for discovering what the server supports and resource schema definitions.
+
+
+## Step 1 : Pre-requisites for accessing the Vendasta's SCIM APIs
+
+### 1. Namespace
+
+This must be the id of the partner that wish to manage users in sync with Vendasta. It should be unique for a partner.
```

Comment:
We need to add the details of where they can get this namespace

Reply:
Yes. Do not link internal docs here. External user will not be able to access it.